app/models/concerns/hostext/search.rb

```diff
@@ -205,6 +220,15 @@ def param_conditions(p)
         end
         conditions.empty? ? [] : "( #{conditions.join(' OR ')} )"
       end
+
+      #override these if needed to add connection in plugin
+      def proxy_connections_columns
+        ['subnets.dhcp_id', 'subnets.dns_id', 'subnets.tftp_id', 'domains.dns_id', 'realms.realm_proxy_id', 'hosts.puppet_proxy_id', 'hosts.puppet_ca_proxy_id']
+      end
+
+      def proxy_connections_tables
+        [:subnet, :domain, :realm]
```

Comment:
These joins seem to be happening through the primary interface _only_:

<pre>
>   Host::Managed.joins(:subnet)
2016-02-17T14:10:08 [sql] [D]   Host::Managed Load (0.8ms)  SELECT "hosts".* FROM "hosts" INNER JOIN "nics" ON "nics"."host_id" = "hosts"."id" AND "nics"."primary" = 't' INNER JOIN "subnets" ON "subnets"."id" = "nics"."subnet_id" WHERE "hosts"."type" IN ('Host::Managed')
</pre>

The eager load should probably be `:realm, :interfaces => [:subnet, :domain]` or similar.